src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -252,19 +274,109 @@ void Engine::ApplyPerformanceHints(std::map<std::string, std::string> &config, c
     ngraphFunc->set_rt_info(hints_props, "intel_cpu_hints_config");
 
     const auto perf_hint_name = getPerfHintName();
-    if (perf_hint_name == CONFIG_VALUE(LATENCY)) {
-        config[CONFIG_KEY(CPU_THROUGHPUT_STREAMS)] = latency_hints.first;
-        config[ov::num_streams.name()] = latency_hints.second;
-    } else if (perf_hint_name == CONFIG_VALUE(THROUGHPUT)) {
-        config[CONFIG_KEY(CPU_THROUGHPUT_STREAMS)] = tput_hints.first;
-        config[ov::num_streams.name()] = tput_hints.first;
-        config[CONFIG_KEY_INTERNAL(BIG_CORE_STREAMS)] = std::to_string(tput_hints.second.big_core_streams);
-        config[CONFIG_KEY_INTERNAL(SMALL_CORE_STREAMS)] = std::to_string(tput_hints.second.small_core_streams);
-        config[CONFIG_KEY_INTERNAL(THREADS_PER_STREAM_BIG)] = std::to_string(tput_hints.second.threads_per_stream_big);
+    if (cpuMapAvailable()) {
+        int streams = engConfig.streamExecutorConfig._streams;
+        if (perf_hint_name == CONFIG_VALUE(LATENCY)) {
+            streams = static_cast<int>(getAvailableNUMANodes().size());
```

Comment:
Let's keep `nstreams = 1` for latency mode on two socket platform as before first.

Comparing profiling result of 1 or 2 streams on two socket platform to get the default setting for latency mode is separated task.

Reply:
nstreams = 2 in old logic on 2 sockets machine.